.github/ISSUE_TEMPLATE/pt-study-group.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Forms to create a Study Group
+about: Fill in this form to create a Study Group on Discord Server
+title: 01/22/22 - Solicitação para criação de um Grupo de Estudos - Open Science Labs
```

Comment:
this title will be generated automatically ... so we shouldn't have something fixed
maybe you change it to:

```
title: "[DATE]: [FEATURE NAME] <STUDY GROUP NAME>" 
```